.github/ISSUE_TEMPLATE/gitcoin-bounty-issue.md

```diff
@@ -0,0 +1,21 @@
+---
+name: New Gitcoin Bounty Issue
+about: Use this template to propose new Gitcoin bounties.
+title: "BOUNTY: [Bounty Title]"
+labels: bounty
```

Comment:
Changing this to `dog` since we've been using the `bounty` label only when it's been approved.